src/Umbraco.Cms.Api.Management/Factories/MemberPresentationFactory.cs

```diff
@@ -108,4 +116,24 @@ private async Task<MemberResponseModel> RemoveSensitiveDataAsync(IMember member,
 
         return responseModel;
     }
+
+    private MemberKind GetMemberKind(Guid key)
```

Comment:
I find it very dangerous that we do not persist the memberkind but infer it from configuration.

Reply:
Hm well. This is how the Delivery API auth works 🤷

Reply:
Doesn't make sense to me to have Configuration linked to stored data..., but I am not going to block the PR for this.